ocs_ci/ocs/acm/acm.py

```diff
@@ -41,19 +44,35 @@ def import_cluster_ui(self, cluster_name, kubeconfig_location):
 
         """
         self.navigate_clusters_page()
-        self.do_click(self.page_nav["Import_cluster"])
+        time.sleep(40)
+        self.driver.find_element_by_id("importCluster").click()
+        log.info("Clicked on Import cluster")
+        time.sleep(60)
         self.do_send_keys(
             self.page_nav["Import_cluster_enter_name"], text=f"{cluster_name}"
         )
+        log.info("Keys sent")
+        log.info("sent input to the text box")
         self.do_click(self.page_nav["Import_mode"])
         self.do_click(self.page_nav["choose_kubeconfig"])
         log.info(f"Coping Kubeconfig {kubeconfig_location}")
         kubeconfig_to_import = copy_kubeconfig(kubeconfig_location)
         for line in kubeconfig_to_import:
-            self.do_send_keys(self.page_nav["Kubeconfig_text"], text=f"{line}")
-            time.sleep(2)
+            if len(line) > 100:
+                for c in line:
+                    self.do_send_keys(self.page_nav["Kubeconfig_text"], text=f"{c}")
+            else:
+                self.do_send_keys(self.page_nav["Kubeconfig_text"], text=f"{line}")
+            time.sleep(10)
```

Comment:
What are we waiting for here? If possible, use explicit wait

Reply:
@am-agrawa , same selenium gotcha, element found would return true but nothing works hence sleep

Reply:
@am-agrawa , I looked at it again, here we are sleeping after sending texts to kubeconfig text box. I am not sure on what we can explicitly wait here, because at any point in the text box there will be partial text available and comparing it with full text is not really a fool proof action. So I feel a sleep here would do better.  WDYT?

Reply:
Right, this was a workaround we came up with to avoid browser crash while copying kubeconfig at once. 
I had a slightly different suggestion though, something like this- 
```
for line in kubeconfig_to_import:
            if len(line) > 100:
                for c in line:
                    self.do_send_keys(self.page_nav["Kubeconfig_text"], text=f"{c}")
                    time.sleep(0.5)
            else:
                self.do_send_keys(self.page_nav["Kubeconfig_text"], text=f"{line}")
           
```
Let's keep it if it's working fine.

Reply:
Had a call with Sylesh to find an alternative not to send char by char as it's taking hours to send the complete kubeconfig. Rather, send it line by line and use a small wait after sending every line.